<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunePickDialog</class>
 <widget class="QDialog" name="TunePickDialog">
  <property name="windowTitle">
   <string>Choose alarm tone</string>
  </property>
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <widget class="ExpandingListWidget" name="tuneList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingListWidget</class>
   <extends>QListWidget</extends>
   <header>ExpandingListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
